api-helpers/prisma/schema.prisma

```diff
@@ -65,8 +65,7 @@ model Blog {
 
   articles Article[]
   
-  addedById String
-  addedBy User @relation(fields: [addedById], references: [id])
+  isPublic Boolean @default(false)
```

Comment:
isPublic? :/

Może `isDraft` lub `isAccepted`?

Reply:
Co jest nie tak z isPublic?

Reply:
Czepiam się. Nazwałbym to inaczej, chociaż patrząc na efekt może mieć to sens